etc/testing/migration/v1_7/diagonal.sh

```diff
@@ -14,7 +14,6 @@
 # process commits with no matching datums)
 
 HERE="$(dirname "${0}")"
-# shellcheck source=./etc/testing/migration/v1_7/deploy.sh
```

Comment:
Removal of this will likely cause shellcheck on CI to fail, but LGTM if it passes

Reply:
I see, so I removed this line because it was causing shellcheck to fail on my local machine. I'll add it back in I guess, passing on CI is more important than my local machine.

Reply:
It should pass on your local machine, so long as you're on shellcheck 0.7 or newer